cmd/ingester/main.go

```diff
@@ -47,9 +47,9 @@ func main() {
 
 	v := viper.New()
 	command := &cobra.Command{
-		Use:   "(experimental) jaeger-ingester",
```

Comment:
The use should not include spaces otherwise generated files look like:

```
(experimental).md
(experimental)_docs.md
(experimental)_env.md
(experimental)_version.md
```